### PR TITLE
INT-547 Add namespacing for PatientHistory

### PR DESCRIPTION
**Pull request by michaeldumalag**
Modified and added all identified files similar to the namespacing pattern in order to add PatientHistory as a solution to fix specified bug below.

Bug Ticket: https://qcentrix.atlassian.net/browse/INT-547

### CODE REVIEW

**Comment by JustinHallquist** [comment]
how did you go about testing these changes?

**Comment by michaeldumalag** [comment]
@JustinHallquist Tested through web aka rails c

**Review by JustinHallquist — COMMENTED** [review]
Could you verify that the changes in
lib/health-data-standards/models/record.rb 
and the added json files are necessary for updating the namespace please

**Comment by michaeldumalag** [comment]
Verified the addition to record schema and related associations are not required for this bug.
@JustinHallquist changes made.

**Review by JustinHallquist — APPROVED** [review]
👍

File: lib/health-data-standards/models/patient_history.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class PatientHistory < Entry
```
Comment (dillonwelch) [inline lib/health-data-standards/models/patient_history.rb:1]: What is the purpose of this model

**Comment by dillonwelch** [comment]
All changes to HDS/QME need PRs on ecqm-api and web to confirm that they do not introduce breaking changes.

File: lib/health-data-standards/models/patient_history.rb (hunk 1 above)
Comment (michaeldumalag) [reply lib/health-data-standards/models/patient_history.rb:1]: This was in order to get namespacing identical to FamilyHistory and Visit::FamilyHistory.  The patern was matched from FamilyHistory in Health-Data-Standards

**Comment by JustinHallquist** [comment]
Do we have an outlined process for dealing with these namespace changes or is there a place I can put one to gather quickly?

File: lib/health-data-standards/models/patient_history.rb (hunk 1 above)
Comment (dillonwelch) [reply lib/health-data-standards/models/patient_history.rb:1]: This isn't adding any namespacing so I don't understand why it's necessary.

**Comment by dillonwelch** [comment]
Nope :trollface: 

You can look at previously merged PRs in this repo

**Comment by JustinHallquist** [comment]
Wouldn't provide clarity around this point though 😢 
```
All changes to HDS/QME need PRs on ecqm-api and web to confirm that they do not introduce breaking changes.
```

File: lib/health-data-standards/models/patient_history.rb (hunk 1 above)
Comment (JustinHallquist) [reply lib/health-data-standards/models/patient_history.rb:1]: Will agree that it seems suspicious

**Comment by michaeldumalag** [comment]
Closing, PR not necessary

**Pull request closed without merging**